recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
     - 0004-CVE-2017-8872.patch
 
 build:
-  number: 2
+  number: 3
```

Comment:
@traversaro I'm no Windows expert but should we mark build numbers 1 and 2 as broken?

Reply:
They are broken only if you want to use them to build somethind consuming the .pc file, but yes probably this is a good idea indeed. See https://github.com/conda-forge/admin-requests/pull/456 .